lorenz96.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+from mpl_toolkits.mplot3d import Axes3D
+from scipy.integrate import odeint
+import matplotlib.pyplot as plt
+
+
+def lorenz96(x, t, N=5, F=8):
+    """
+    This is the Lorenz 96 model with constant forcing.
+    Code snippet adapted from a Wikipedia example.
+    Here the differential equation is described for
+    use with the ``odeint`` function.
+
+    Parameters:
+    -----------
+    x : The independent variable
+    t : The time of the simulation
+    N : integer
+        The number of variables in the system
```

Comment:
N >= 4 is a constraint for generate_L96. Is it also a constraint for this function?

Reply:
It's not really a constraint. The model is still defined for N < 4, it's just less interesting. Fixed the doc string